src/include/graph/graph.h

```diff
@@ -5,6 +5,7 @@
 #include <memory>
 #include <optional>
 
+#include "catalog/catalog_entry/table_catalog_entry.h"
```

Comment:
Can you double check if you can turn this include into a forward declaration?

Reply:
Done